### DISCUSSION TITLE
Idea: Mass Delete Sprites/Costumes etc.

### DISCUSSION THREAD

**Discussion by ArcherThesu** [ScratchAddons/ScratchAddons#5183] [Ideas]
Basically the title. I've needed this feature several times for some reason and I haven't seen anything or anyone suggesting or implementing this.

**Comment by WorldLanguages** [comment]
Looks like #4692 for costumes

**Comment by DNin01** [comment]
Do you mean a feature where you can multi-select things and then delete them all at once?

**Reply by penguinmoose** [reply]
I think that would be better - you can hold shift and click multiple costumes or sounds and also use arrow keys to control the selection. It would allow for easy deleting of multiple, but not all, costumes/sounds as well as making it much easier to delete everything.

**Reply by ArcherThesu** [reply]
yeah multi-selecting basically
